jobrunner/secrets-template.env

```diff
@@ -0,0 +1,22 @@
+# These are secrets that should not be kept privately
+
+# Credentials for logging into the job server
+JOB_SERVER_TOKEN=
+
+# A Github developer token that has read access to private repos
+PRIVATE_REPO_ACCESS_TOKEN=
+
+# A database containing dummy data; results from this could be freely
+# published without review
+DUMMY_DATABASE_URL=
```

Comment:
We've basically decided to get rid of this. We're no longer running the dummy TPP SQL Server, I *think* it's no longer an option in the job server, and we're going to support custom expectations-based dummy backends at Some Point

Reply:
Ok, will drop it.

I have a vague plan for better config documentation and validation, but it's not top priority